config/routes.rb

```diff
@@ -2,6 +2,7 @@
   root "home#index"
 
   get "resources", to: redirect("/admin")
+  post "/rails/active_storage/direct_uploads", to: "/active_storage/direct_uploads#create"
```

Comment:
This was generating a weird route nested under `/avo`

```diff
--  post "/rails/active_storage/direct_uploads", to: "active_storage/direct_uploads#create"
++  post "/rails/active_storage/direct_uploads", to: "/active_storage/direct_uploads#create"
```

![image](https://user-images.githubusercontent.com/1334409/148254990-245948f5-d3fd-47d7-9c87-84a69f3a80f7.png)

Reply:
Also, we don't need the `/avo` route because it will all be nested under `/avo` (or whatever `/admin` route is used).